Added contact form

wajahatalli opened the pull request:
# Changes in this Pull Request

- Added h1 tag in the Contact form
- Added description in the Contact form
- Added the contact form 
- Added styles to it, to make it look like a Figma template
- Added a way to receive the Forms submitted by others
- changed the type="button" to type="submit"
- used one font style across the whole HTML
- Made required changes in COntact Form design.

anagudelogu reviewed (changes requested):
Hi team! @wajahatalli @Ellon-M, 

Great job working on your contact section! 🍾 🎆 
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media0.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights
✅ Professional and descriptive PR title and summary. 💯 
✅ Great use of GitHub flow. 👍🏼 
✅ Meaningful and descriptive commit messages. 🥇 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wajahatalli commented:
1. Added a way to receive the Forms submitted by others
2. changed the type="button" to type="submit"
3. used one font style across the whole HTML
4. Made required changes in Contact Form design.

Alfredbis29 commented on index.html:
```diff
@@ -1,257 +1,272 @@
+        <small id="message-success" class="hidden">From Submitted</small>
+        <input type="submit" class="btncontact" value="Get It Touch" />
+      </form>
+      <div class="contact-bar">
+        <img class="end-line" src="assets/images/Shape.png" alt=" end line icon" />
+      </div>
```
- [ OPTIONAL] I can see that you are using `img` tag to put line bar, on my opinion this cannot be an `img` tag, I can suggest you to use a `div` tag then give it a class name and design it in your style file  :wink:

wajahatalli replied:
Okay. thank you, i'll consider.

Alfredbis29 reviewed (approved):
# STATUS: APPROVED  
 ## Hi @wajahatalli ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:

![](https://camo.githubusercontent.com/54a5d803a138b5a1a1242a607e4c1192c852cab8c591c1ddcaa33d06b51560f6/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f686c516559335062684833692f67697068792e676966)
 
 ## Optional suggestions

- [ ] Check below the review 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
